src/routes/tools/extended-drought/_ExploreData.svelte

```diff
@@ -97,7 +98,7 @@
 
   async function loadShare() {
     if ($boundary.id === "custom") {
-      bookmark = "Cannot create a bookmark for an uploaded boundary";
+      shareLinkWarning = "Cannot create a bookmark for an uploaded boundary";
```

Comment:
Maybe make the warning text say "share link" instead of "bookmark" because the modal's button is called "share"